OpenCL_C.txt

```diff
@@ -10822,20 +10820,21 @@ events.
       Otherwise returns _false_.
 | void *set_user_event_status*(clk_event_t _event_, int _status_)
     | Sets the execution status of a user event.
-      _event_ must be a user-event.
+      Behavior is undefined if _event_ is not a valid event returned by
+      *create_user_event*.
       _status_ can be either `CL_COMPLETE` or a negative integer value
       indicating an error.
 | |
 | void *capture_event_profiling_info*(clk_event_t _event_,
   clk_profiling_info _name_, global void *_value_)
    a| Captures the profiling information for functions that are enqueued as
       commands.
-      The specific function being referred to is: enqueue_kernel.
       These enqueued commands are identified by unique event objects.
       The profiling information will be available in _value_ once the
       command identified by _event_ has completed.
 
-_event_ must be an event returned by enqueue_kernel.
+      Behavior is undefined if _event_ is not a valid event returned by
+      *enqueue_kernel*.
```

Comment:
```suggestion
Behavior is undefined if _event_ is not a valid event returned by
*enqueue_kernel*.
```

Having this sentence indented causes it to be rendered in a box because of the `a` [cell specifier](https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/) on the cell.

Removing the specifier breaks the `NOTE` at the bottom of the cell.